manual/python/index.md

```diff
@@ -282,8 +282,7 @@ with the necessary information on how to generate Python bindings on the fly.
 This is what the large LHC experiments do to steer their analysis frameworks from Python.
 
 1. Create one or multiple C++ libraries. <br/>For example create a library as a CMake project that uses ROOT, see → [CMake details]({{ '/manual/integrate_root_into_my_cmake_project' | relative_url }})
-2. [Optional] Add [`ClassDef` macros]({{ 'manual/root_io/#adding-a-class-to-root' | relative_url }}) for classes that should be read or written from or into files.
```

Comment:
ClassDef is still useful (allows for faster operations and for embedding the version number with the class declaration rather than the LinkDef file), so I am not convinced we should hide/remove it from the doc.

Reply:
This removes it from `manual/python/`. This whole section will just become a pointer to root.cern/manual/io_custom_classes , in fact.

The I/O part of the manual (root.cern/manual/io_custom_classes , root.cern/manual/root_files etc.) should absolutely mention `ClassDef`.

Reply:
I missed the context.  Re-reading, I see 
```
This is what the large LHC experiments do....
... Use a [`LinkDef.h` file]...
```
Technically most LHC experiement framework use `selection.xml` files (and `genreflex` (now) or `rootcling --genreflex` when enabling runtime_module for their libraries) [And users that do add to the framework are exposed to that (they have to update the selection.xml)

Reply:
if you want to suggest changes to `manual/python` please mention this at https://github.com/root-project/web/pull/726 or https://mattermost.web.cern.ch/root/channels/pyroot

Reply:
ok.  this wording seems to have been removed (replaced by a pointer to dictionary generation)